src/pages/ProfilePage/Account.js

```diff
@@ -76,10 +77,13 @@ class ProfilePage extends React.Component {
 
 	componentDidMount() {
 		const {
-			usecase, deploymentTimeframe, phone, company, username, picture,
+			usecase, deploymentTimeframe, phone, company, username, picture, isShopify,
 		} = this.props; // prettier-ignore
+
+		const defaultUseCase = isShopify ? useCaseOptions[2] : usecase;
+
 		this.profileForm.patchValue({
-			usecase,
+			usecase: usecase || defaultUseCase,
```

Comment:
@jyash97 I don't think we need any change here.
It's the value coming from API.

Reply:
okay fixing

Reply:
I checked this we will need this because on new Signup backend doesn't have value and when the context is Shopify we want the default value to set is `Ecommerce Store` so I have done this.